plugins/inputs/jolokia2/README.md

```diff
@@ -205,3 +205,7 @@ configurations that apply to every `metric` declaration.
 
 Please help improve this list and contribute new configuration files by opening
 an issue or pull request.
+
+## Metrics
+
+## Example Output
```

Comment:
Empty sections don't do anything but make the readme linter happy. I'd rather leave them out and ignore the readme linter errors until we have content for them.

Reply:
I agree. Will undo the change.